auth/user_mgt_test.go

```diff
@@ -643,6 +643,31 @@ func TestHTTPError(t *testing.T) {
 	}
 }
 
+func TestVersionHeader(t *testing.T) {
```

Comment:
Can we try to test this without a new test case? See my comment below.

Reply:
done